docs/book/index.rst

```diff
@@ -96,6 +96,26 @@ You should also have a look here if you are looking for Cart, which is in Sylius
 
 .. include:: /book/orders/map.rst.inc
 
+API
+---
+
+.. warning::
+
+    The new, unified Sylius API is still under development, that's why the whole ``ApiBundle`` is tagged with ``@experimental``.
+    This means that all code from ``ApiBundle`` is excluded from :doc:`Backward Compatibility Promise </book/organization/backward-compatibility-promise>`.
+
+This chapter will explain you how to start with our new API, show concepts used in it, and you will inform why we decided to rebuild entire api from scratch.
```

Comment:
I am thinking if impersonal form would be better:
```suggestion
This chapter will explain how to start with our new API, show concepts used in it, and will inform why we decided to rebuild entire api from scratch.
```

If we stay like it is now: 
```suggestion
This chapter will explain to you how to start with our new API, show concepts used in it, and will inform you, why we have decided to rebuild entire api from scratch.
```